Resources/doc/empty-content-status-code.rst

```diff
@@ -0,0 +1,27 @@
+Status code when responding with no content
+===========================================
+
+In some use cases the api should not send any content, especially when deleting (*DELETE*) or updating (*PUT* or *PATCH*) a resource.
+
+In order to provide a status code that should be sent when having no content, the configuration must be customized:
+
+.. code-block:: yaml
+
+    fost_rest:
+        view:
+            empty_content: 204
+
+If a controller returns no respone, this will be the status code.
+
+.. versionadded:: 2.0
+  Until FOSRestBundle 2.0 this code will be used even if another code is configured inside the view object!
+
+Changes in 2.0
```

Comment:
but I think that the ability to pass custom status codes even if the content is empty should be part of the documentation